spec/spec_helper.rb

```diff
@@ -33,6 +33,21 @@ def alice
   @alice ||= Person.find_by(diaspora_id: "alice@localhost:3000")
 end
 
+def test_pkey
+  DiasporaFederation.callbacks.trigger(:get_private_key_by_id)
+end
+
+#
+# Sort hash according to an entity class's property sequence.
+# This is used for rspec tests in order to generate correct input hash to
+# compare results with.
+#
+def sort_hash!(data, klass=described_class)
```

Comment:
this is only used in `factories.rb`, so you can move it there.

Also you can remove the `!`, we don't need the `data.replace()`, you can return a new hash.